docs/ai/conceptual/understanding-tokens.md

```diff
@@ -33,8 +33,6 @@ This text could be tokenized as:
 - a
 - cat
 
-The model will add only the first occurrence of the "a" token to its vocabulary. All other occurrences can be represented by that first token.
```

Comment:
Removed this line to simplify the section since it wasn't core to it and it is also described later in the doc with the example given for adding IDs to tokens.